src/corppa/poetry_detection/refmatcha.py

```diff
@@ -1,18 +1,27 @@
 #!/usr/bin/env python
 """
 
-🎶🍵 matcha matcha poem / This script is gon / na find your poems / matcha matcha poem 🎶🍵
+*🎶🍵 matcha matcha poem / This script is gon / na find your poems / matcha matcha poem 🎶🍵*
 
-refmatcha is a script to identify poem excerpts by matching against a local
-collection of reference poems.  It takes in a CSV of excerpts
-and outputs a CSV of labeled excerpts for those excerpts it is able to identify.
+refmatcha identifies poem excerpts by matching against a local collection of
+reference poems.  It takes in a CSV of excerpts and outputs a CSV of
```

Comment:
```suggestion
reference poems.  It takes in a CSV of detected (i.e. unidentified) excerpts and outputs a CSV of
```
An alternative could be explicitly mention a CSV of poetry_detection.core.Excerpt data